slides/04-networking-in-riot/networking-in-riot.md

```diff
@@ -450,6 +464,40 @@ Follow the instructions in the notebook **riot/networking/coap/coap.ipynb**
 
 ---
 
+## Bonus: BLE
```

Comment:
I'm fine with adding this here for now but I think in the future it would make more sense to add a section dedicated to BLE with more exercises and a more detailed description of BLE in general.

Reply:
I agree, we are adding it a bit because we now most participants are interested in BLE (Emmanuel has some inside information on people joining the Inria Academy session), and also to test it out. There is definitively much more we can do with BLE (IpV6 over BLE, RPL, GATT, etc..)